modules/gapi/samples/pipeline_modeling_tool/pipeline_builder.hpp

```diff
@@ -258,6 +258,7 @@ struct InferParams {
     std::vector<std::string> input_layers;
     std::vector<std::string> output_layers;
     std::map<std::string, std::string> config;
+    cv::util::optional<int> out_precision;
```

Comment:
So here it is int, not a map of ints?

Reply:
Config has parameter `output_precision` which is applied for all output layers. There is no need to have per layer precision (so far...)